src/components/Rockets/Rocket.jsx

```diff
@@ -0,0 +1,18 @@
+const Rocket = ({description, title, image, id}) => {
+  
+    return (
+      <div>
+        <div>
+          <img src={image} alt="" />
```

Comment:
- [ ] For browser accessibility please add an alternative text to the rocket image, you can get it from `{title}`.